compiler/lexer.cpp

```diff
@@ -58,6 +58,7 @@ static cell litchar(const unsigned char** lptr, int flags);
 
 static void substallpatterns(unsigned char* line, int buffersize);
 static int alpha(char c);
+static void set_file_defines(char* file);

```

Comment:
spcomp is C++ now, and uses STL, so we can make this a bit prettier:

`static void set_file_defines(char* file);`

Reply:
changed the signature to `static void set_file_defines(std::string file);`